data/simulation/config.default.yaml

```diff
@@ -42,29 +42,54 @@ tx-max-size-bytes: 16384
 # Ranking Block Configuration
 ################################################################################
 
+# 1/leios-stage-length-slots, targeting one RB per pipeline.
+# Also 20s is current rate of praos blocks.
 rb-generation-probability: 5.0e-2
-rb-generation-cpu-time-ms: 300.0
-rb-head-validation-cpu-time-ms: 1.0
-rb-head-size-bytes: 32
+# Eng. team targets 1kB as worst case upper bound.
+# Actual size fairly close.
+rb-head-size-bytes: 1024
 rb-body-max-size-bytes: 90112
+# Note: certificate generation/validation is not included in the
+# timings here, see cert-* fields.
+rb-generation-cpu-time-ms: 1.0
+rb-head-validation-cpu-time-ms: 1.0
 
+# On average, no Txs directly embedded in blocks.
+rb-body-legacy-praos-payload-avg-size-bytes: 0
 rb-body-legacy-praos-payload-validation-cpu-time-ms-constant: 50.0
+# the -per-byte component is meant to be using size as a (bad)
+# proxy for the complexity of the Txs included.
 rb-body-legacy-praos-payload-validation-cpu-time-ms-per-byte: 0.0005
-rb-body-legacy-praos-payload-avg-size-bytes: 0
 
 ################################################################################
 # Input Block Configuration
 ################################################################################
 
 ib-generation-probability: 5.0
-ib-generation-cpu-time-ms: 300.0
 ib-shards: 1
-ib-head-size-bytes: 32
+
+# ProducerId  32
+# SlotNo      64
+# VRF proof   80
+# Body hash   32
+# RB Ref      32
+# Signature   64
+# Total       304
+#
+# NOTE: using a KES Signature (like for Praos headers)
+#       would instead more than double the total to 668.
+#       And even 828 including Op Cert.
+ib-head-size-bytes: 304
```

Comment:
@pagio do you believe we want KES signatures in IB headers and EBs, or something smaller would be enough?